storage/store/sql/specific_sqlite.go

```diff
@@ -81,5 +81,27 @@ func (s *Store) createSQLiteSchema() error {
 	`)
 	// Silent table modifications TODO: Remove this in v6.0.0
 	_, _ = s.db.Exec(`ALTER TABLE endpoint_results ADD domain_expiration INTEGER NOT NULL DEFAULT 0`)
+
```

Comment:
Can you move the code you added above the `// Silent table modification` comment? Maybe add in a comment about how this improves the perf too above the block of code you're adding 👀